packages/spectral/src/testing.ts

```diff
@@ -477,6 +478,7 @@ export class ComponentTestHarness<TComponent extends Component> {
     return result;
   }
 
+  /** Invoke a trigger by its key within a unit test */
   public async trigger<TConfigVars extends ConfigVarResultCollection>(
```

Comment:
No JSDocs blocks for these guys?

Reply:
I must have missed this file; good catch!